tests/baselines/reference/privateIndexer2.errors.txt

```diff
@@ -13,7 +13,7 @@ tests/cases/conformance/classes/indexMemberDeclarations/privateIndexer2.ts(4,32)
                   ~
 !!! error TS1005: ']' expected.
                     ~~~~~~
-!!! error TS2304: Cannot find name 'string'.
+!!! error TS2552: Cannot find name 'string'. Did you mean 'String'?

```

Comment:
Feels like we need to ban some global identifiers but not others. The boxed primitives, null, and NaN in particular... probably the rest can stay?